src/maker/handlers.rs

```diff
@@ -32,7 +32,7 @@ use crate::{
 
 use super::{
     error::MakerError,
-    maker::{ConnectionState, Maker, MakerBehavior},
+    server::{ConnectionState, Maker, MakerBehavior},
 };
```

Comment:
We can consolidate these two imports into one single `crate` level import. No need for `super`. This should have been done before, so might as well do it here.

Reply:
Like [this](https://github.com/utxo-teleport/teleport-transactions/pull/50/commits/2ba960a628cc6b6b0004ab36e728af4e1c4b0d9a)?